### ISSUE TITLE
Invoice error: 'One or more fields is invalid'

### ISSUE THREAD

**Issue by DropaCloud** [bitpay/whmcs-plugin#28]
Hi,

I am facing new error with bitpay plugin, help to solve.
[ERROR] In modules/gateways/bitpay/createinvoice.php: Invoice error: 'One or more fields is invalid'

There is no error log produced in whmcs.

waiting for your reply.

**Comment by kleetus** [comment]
Would you kindly reply with the following:

1. the WHMCS version
2. the php version
3. the full request that was made to produce the error

thanks

**Comment by TheMaskedMonkey** [comment]
I literally just resolved this issue myself. I looked at the headers like Kleetus said and found that WHMCS was sending me to HTTP and then Cloudflare redirecting to HTTPS as it should therefor not sending the correct data to the PHP file. The way I fixed this was to send the System URL to the https://domain.com. If I set it to the HTTP version and then the SSL version to HTTPS I end up getting a redirect loop so that's a quick way to fix it ;) - I hope this helps!
